### ISSUE TITLE
Custom Block support

### ISSUE THREAD

**Issue by Nerade** [World-of-Tomorrow/NoItem#91]
Hi,

is it possible to reactivate this feature manually or is the reason for disabling it severe? I really like your plugin and i'm kinda dependent on it. I'm experienced in programming plugins, but what is the right way to re-implement this genius feature?

Thanks alot

**Comment by just-push-it** [comment]
It depends on what way you are adding custom blocks, whether is be with Forge or something else. The old custom blocks support was built specifically for Tekkit, but when they stopped updating it, I had no reason to continue supporting it in new releases.

**Comment by Nerade** [comment]
We have a system based on MCPC+ and the new Technic Launcher. With this combination we have the opportunity to build a modpack equal to Tekkit but updated to 1.4.7. Is it possible to adapt the Tekkit version to 1.4.7 MCPC+? Thanks for your fast reply.

**Comment by Nerade** [comment]
Are you so kind to tell me if that is a reason for you to reimplement this feature? I just need an answer...this would be great! =) Otherwise I will try this on my own. Sorry for the inconvenience so far.

**Comment by just-push-it** [comment]
Is MCPC+ something that is frequently and rather quickly updated with Minecraft?

**Comment by Nerade** [comment]
As far as I can say the are up-to-date with minecraft and forge at least since 1.2.5. Currently they are on minecraft 1.5.1. So I guess yes. But my first tries regarding the requested feature showed my that there are no events which will be fired so far...(onInventoryClick). Maybe you are more in detail but I guess this is necessary?!

**Comment by cheekyvimto** [comment]
I'm also using MCPC+ which is the to my knowledge the most active/up-to-date forge/bukkit bridging project.
I think all that is required to make this plugin compatible is to allow for adding additional block/item ids.
Forge adds the ability to go upto id 4096 for custom blocks (smallint?)
If you can make NoItem do this, I think you will attract a large forge based admin following, I know I need this feature to pull off what I am trying to achieve.

Is this something you would consider in a future update?

**Comment by just-push-it** [comment]
Yes, it is definitely something I would add support for in the future if it is a viable option. I will look into it when I get time and am finished with the current update I'm working on!

**Comment by cheekyvimto** [comment]
That's great to hear, I'd like to have NoItem as a cornerstone plugin for our server!
